.idea/community.iml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we need all this `.idea/` stuff?

Reply:
Creating an issue for it (gitignore)

Reply:
I don't think so. Editor added files I guess. :/